package.json

```diff
@@ -57,6 +57,9 @@
   ],
   "author": "Joel Denning",
   "license": "MIT",
+  "peerDependencies": {
```

Comment:
The state of peerDependencies across all package managers is so complicated that I worry that this would require making this a major version change to single-spa.